src/reanimated2/mock.js

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable standard/no-callback-literal */
+const { Easing } = require('./Easing');
+
+const NOOP = () => {};
+
+const ReanimatedV2 = {
+  useSharedValue: (value) => ({ value }),
+  useDerivedValue: (a) => ({ value: a() }),
+  useAnimatedScrollHandler: () => NOOP,
+  useAnimatedGestureHandler: () => NOOP,
+  useAnimatedStyle: (style) => style,
+  useAnimatedRef: () => ({ current: null }),
+  useAnimatedReaction: NOOP,
+
+  withTiming: (toValue, config, cb) => {
+    cb && setTimeout(() => cb(true), config?.duration ?? 300);
```

Comment:
Why only timing has a duration that isn't 0? And it shouldn't be somehow synchronized with default values in JS code?

Reply:
My initial thoughts on this are that since timing is the only one with a user-specified duration, respecting that duration may be the right choice and the expected behavior by the user.

But I not so sure about it. I can change it to 0.

Reply:
@Szymon20000, @karol-bisztyga do you have any thoughts about this?

Reply:
All right, seems like no one had time to look at it. For me, it should have duration 0, we don't want to artificially delay tests.

Besides, is `setTimeout` necessary? Can't we just call the callback directly?

Reply:
Fixed in c6807ec.